src/utils/gwhisperUtils.cpp

```diff
@@ -0,0 +1,31 @@
+#include "gwhisperUtils.hpp"
+#include <fstream>
+namespace gwhisper
+{
+    namespace util
+    {
+
+        /// Utility-Function for reading contents of a file and returning them as a string
+        /// This function terminates the program, if file not found
+        /// @param f_path Location of the file to read from as a string
+        /// @return File content as simple string
+        std::string readFromFile(const std::string f_path)
+        {
+            std::ifstream credFile(f_path);
+            const char *file = f_path.c_str();
+            if (file)
```

Comment:
what are you checking here? this is implicit conversion to bool and should be avoided. Explicitly use comparison operators.
Afaik you are checking if the filename converted into a c_str is not a nullptr or not. This will never be false.